### PR TITLE
New package: pcem-17

### PR DESCRIPTION
**Pull request by ajshell1**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/pcem/INSTALL.msg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+pcem stores BIOS ROM images (not included),
+configuration files, and other data in ~/.pcem.
```
Comment (ericonr) [inline srcpkgs/pcem/INSTALL.msg:2]: This is documentation for the package and I don't believe it should be included in an INSTALL.msg.

File: srcpkgs/pcem/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+short_desc="Emulator for old x86 PCs"
+maintainer="ajshell1 <ajshell1@protonmail.com>"
+license="GPL-2.0-only"
+homepage="https://pcem-emulator.co.uk/"
+distfiles="https://pcem-emulator.co.uk/files/PCemV${version}Linux.tar.gz"
+checksum=5b24cb5ce886ed53232385f46594146ba3f7d7eecda90f82892b2dce1cb2f1a4
+
+pre_configure() {
+autoreconf
+}
```
Comment (ericonr) [inline srcpkgs/pcem/template:20]: Please respect the indentation. And I believe we use the `-fi` flags for autoreconf as well.

File: srcpkgs/pcem/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'pcem'
+pkgname=pcem
+version=17
+revision=1
+wrksrc="${pkgname}-${version}"
+create_wrksrc=yes
```
Comment (ericonr) [inline srcpkgs/pcem/template:6]: If you need `create_wrksrc` you don't have to define `wrksrc`.

File: srcpkgs/pcem/template [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+build_style=gnu-configure
+configure_args="--enable-alsa --enable-release-build --enable-networking --prefix=/usr"
```
Comment (ericonr) [inline srcpkgs/pcem/template:8]: `--prefix=/usr` is default in the build style.

File: srcpkgs/pcem/template (hunk 2 above)
Comment (ajshell1) [reply srcpkgs/pcem/template]: I've discovered that removing this section entirely fixes the compilation on most ARM platforms, so I've chosen to remove it entirely.

**Comment by ajshell1** [comment]
I've just realized that this is a duplicate of  pull request  #22340

I'll admit to blatantly using some of his code to improve my own pull request. For example, I realized that the automake was unnecessary. Also, I dug through pcem's configure file and realized that it only accepts i386/i686, x86_64, armv7l, and aarch64 CPUs. For this reason, I thought it was appropriate to include those specific architectures in the template.

At this point, here are a list of substantial differences between our templates:

1. Mine thinks that the license is GPL2 only, he thinks it's GPL2 or later. I have no idea which is right.
2. Somewhat different method of assigning the distfiles variable
3. He uses do_extract() while I use create_wkspace, to the same effect.
4. He uses vdoc on the readme files and tested files.
5. Different short_desc
6. Different maintainer.

I don't particularly care which of us is selected, as long as one of us is.

**Comment by ScrelliCopter** [comment]
Author of the first PR here, just saw this.

In my experience Void usually puts precedence on the first PR in the case of duplicates. I'm not precious about who gets to be maintainer but if you'd like to make sure any improvements over my template (touché on using create_wrksrc) make it in I'd be happy to take reviews on my PR.

It seems you already figured out the reasons behind everything I was doing but I'll elaborate on why I chose `GPL-2.0-or-later`:
Sarah never appeared to have included a copy of the GPL or attached any GPL text to her own code, grepping thru showed a lot of "version 2 (or at your option) any later version"; and actually I just looked thru v17 again and `src/f82c710_upc.c` and `src/sound_azt2316a.c` are version **3** or later, so it seems like the license field should actually be `GPL-3.0-or-later` 🤷.

**Comment by abenson** [comment]
Upstream is dead.  We can re-approach if there's a fork.

> **14th June 2021**
> Just a quick note to say that I (Sarah Walker) have decided to call it quits.

**Pull request closed without merging**